mate-panel/panel.c

```diff
@@ -598,6 +598,109 @@ drop_caja_desktop_uri (PanelWidget *panel,
 	return success;
 }
 
+enum {
+	RESPONSE_SET_BACKGROUND,
+	RESPONSE_CREATE_LAUNCHER
+};
+
+static void
+image_drop_dialog_response (GtkWidget   *dialog,
+			    int          response,
+			    PanelWidget *panel)
+{
+	if (response == RESPONSE_SET_BACKGROUND) {
+		const gchar *uri = g_object_get_data (G_OBJECT (dialog), "uri");
+
+		set_background_image_from_uri (panel->toplevel, uri);
+	} else if (response == RESPONSE_CREATE_LAUNCHER) {
+		const gchar *uri = g_object_get_data (G_OBJECT (dialog), "uri");
+		gint pos = GPOINTER_TO_INT (g_object_get_data (G_OBJECT (dialog), "pos"));
+
+		drop_uri (panel, pos, uri, PANEL_ICON_UNKNOWN);
+	}
+
+	gtk_widget_destroy (dialog);
+}
+
+static GtkWidget *
+image_drop_dialog (PanelToplevel *toplevel,
+		   const gchar   *uri)
```

Comment:
indent

Reply:
@rbuj  the indentation is ok, see https://github.com/mate-desktop/mate-panel/blob/eff98bfc06d0e55f9c375fe9ebd48328b5e1e913/mate-panel/panel.c

it respects old style